src/kernels/raw/session/rawSocket.node.ts

```
@@ -278,6 +278,8 @@ export class RawSocket implements IWebSocketLike, IKernelSocket, IDisposable {
                 this.postToSocket(msg.channel, data);
             });
         }
+        // Ensure we don't have any unhandled exceptions (swallow exceptions as we're not awaiting on this promise).
+        this.sendChain.catch(noop);
```

Swallow more unhandled exceptions for dangling promises